components/formats-bsd/src/loci/formats/tools/PrintDomains.java

```diff
@@ -52,18 +52,13 @@ public static void main(String[] args) {
     // get a list of all available readers
     IFormatReader[] readers = new ImageReader().getReaders();
 
-    Hashtable<String, Vector<IFormatReader>> domains =
-      new Hashtable<String, Vector<IFormatReader>>();
-
-    for (String domain : FormatTools.ALL_DOMAINS) {
-      domains.put(domain, new Vector<IFormatReader>());
-    }
+    final TreeMultimap<String, String> domains = TreeMultimap.create();
```

Comment:
I figured we were just putting a bunch of empty lists in so that we can then add a format to a list without checking if a list is waiting to receive it. (Multimaps handle this chore for us.) I suppose it means that for a domain that had no readers we wouldn't now bother printing the heading with nothing below it before the next heading, but in practice all the domains have a good few readers associated, at least these days. (This can be tried out with `tools/domainlist`.)